src/cozmo_companion/assistant.py

```diff
@@ -51,13 +52,9 @@ class VoiceAssistant:
     """
 
     def __init__(self):
-        self.conversation_history = [
-            {
-                "role": "system",
-                "content": "You are a helpful, friendly assistant.If you detect \
-             negative sentiment from the user, reply with a joke.",
-            }
-        ]
+        self.chatbot = AIApplication(description=("A friendly, supportive chatbot."))
```

Comment:
You can give some more instructions like
- `"if you detect negative sentiments, help the user with some jokes and motivational prep talk"`
- ...